pages/cart-management/cart-management.css

```diff
@@ -51,7 +49,7 @@
   justify-content: center;
   align-items: center;
```

Comment:
the class flex-total-center has the  for display:flex , justify-content-center and align-items:center so maybe we can remove these line from here and use the class ?

Reply:
Yeah, I must have missed that thanks for looking into the code.

Reply:
> the class flex-total-center has the for display:flex , justify-content-center and align-items:center so maybe we can remove these line from here and use the class ?

Yeah, I must have missed that thanks for looking into the code.